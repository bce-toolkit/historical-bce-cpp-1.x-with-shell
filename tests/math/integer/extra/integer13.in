62021639375507579896490351925403947016474703648327494943530025473161965194
412411785222607393611243513397387143655360339456025573000498722459617381098120945237160145436
