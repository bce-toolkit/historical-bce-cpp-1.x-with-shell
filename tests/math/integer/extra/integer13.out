412411785222607393673265152772894723551850691381429520016973426107944876041650970710322110630
-412411785222607393549221874021879563758869987530621625984024018811289886154590919763998180242
25578455017285841795019916443881937137191183122352924973637794646261150052776074485935552822675279941560058967664028691044366125037706533545296525340264710192609954584
0
62021639375507579896490351925403947016474703648327494943530025473161965194
0
1
1
1
0
0
